main.js

```diff
@@ -0,0 +1,66 @@
+console.log('holla from js');
+
+var expenseName = document.forms[0].querySelector('input[name="expense-name"]');
+var expenseAmount = document.forms[0].querySelector('input[name="expense-amount"]');
```

Comment:
No need to go through `.forms[0]` if you're calling `.querySelector` just do it directly on the document.